core/client/src/main/java/alluxio/hadoop/AbstractFileSystem.java

```diff
@@ -91,6 +91,7 @@
    */
   AbstractFileSystem(FileSystem fileSystem) {
     sFileSystem = fileSystem;
+    sInitialized = true;
```

Comment:
`FileSystem.Factory.get()` must be called after `Configuration` initialized (in here `Configuration` may be initialized during `intialize()` methid. So we could not set the `FileSystem` member variable in construction time but lazily.